mailpoet/views/analytics.html

```diff
@@ -5,24 +5,21 @@
 
 window.mixpanelTrackingId = "8cce373b255e5a76fb22d57b85db0c92";
 
-if (mailpoet_analytics_enabled) {
-
-  mixpanel.init(window.mixpanelTrackingId, {
-    loaded: function(mixpanel) {
-      // used in lib/Analytics/Analytics.php
-      document.cookie = "mixpanel_distinct_id=" + mixpanel.get_distinct_id();
-    }
-  });
+mixpanel.init(window.mixpanelTrackingId, {
+  loaded: function(mixpanel) {
+    // used in lib/Analytics/Analytics.php
+    document.cookie = "mixpanel_distinct_id=" + mixpanel.get_distinct_id();
+  }
+});
```

Comment:
Why do we set the cookie still and reference Analytics.php. I think this is not necessary anymore? Or we could change the behavior in a way that we generate a random id only when the cookie does not exist?

Reply:
Do we use mixpanel in JavaScript anywhere? I assume otherwise we could get rid of the analytics.html? I am worried we could get out of sync between JS events and PHP events, but probably not, since we set the public Id in the twig function, right? So I assume we should just remove this setting of the cookie to avoid confusion :thinking:

Reply:
Sounds good to me. Thank you @johnolek